SteakBot.Core/EventHandlers/VoiceStateEventHandler.cs

```diff
@@ -1,33 +1,62 @@
-﻿using System.Linq;
-using System.Threading.Tasks;
-using Discord.WebSocket;
+﻿using Discord.WebSocket;
 using SteakBot.Core.EventHandlers.Abstraction;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
 
 namespace SteakBot.Core.EventHandlers
 {
     internal class VoiceStateEventHandler : IVoiceStateEventHandler
     {
-        public async Task HandleUserVoiceStateUpdatedAsync(SocketUser user, SocketVoiceState leaveState, SocketVoiceState joinState)
+        private const string BendoverwatchMesssageChannel = "bendoverwatch";
+
+        // TODO: get from configuration
+        private readonly IDictionary<string, string> _voiceToMessageChannelMapping = new Dictionary<string, string>
+        {
+            { "shavilandia", "bendoverwatch" },
+            { "starcraft_voice", "zurg_op" },
+            { "random_voice", "parjoli" }
```

Comment:
Please add for completeness:
```suggestion
            { "random_voice", "parjoli" },
            { "play_diablo_here_pliz", "general_gaming" }
```